vehicle:                                #  车辆参数
    main_car:
      id: 0
      frame: "base_footprint"
      length: 1.0
      width: 1.5
      pose_x: 0.0
      pose_y: 0.0
      pose_theta: 0.0
      speed_ori: 1.0
    obs_car1:
      id: 1
      frame: "base_footprint_obs1"
      length: 3.0
      width: 1.6
      pose_x: 60.0
      pose_y: 0.0
      pose_theta: 0.0
      speed_ori: 0.0
    obs_car2:
      id: 2
      frame: "base_footprint_obs2"
      length: 3.0
      width: 1.6
      pose_x: 120.0
      pose_y: 16.0
      pose_theta: 1.0
      speed_ori: 0.0
    obs_car3:
      id: 3
      frame: "base_footprint_obs3"
      length: 3.0
      width: 1.6
      pose_x: 85.0
      pose_y: 4.0
      pose_theta: 0.0
      speed_ori: 0.0

pnc_map:                                #  pnc参数
  frame:  "map"
  type: 3
  road_length: 250.0
  road_half_width: 4.0
  segment_len:  0.5
  speed_limit: 1.0

global_path:                            #  全局路径参数
  type: 0

local_path:                             #  局部路径参数
  curve_type: 2
  path_size: 80

local_speed:                            #  局部速度参数
  speed_size: 100

reference_line:                         #  参考线参数
  type: 0
  front_size: 200
  back_size: 80

decision:                               #  决策参数
  safe_dis_l: 0.5
  safe_dis_s: 0.5

planning_process:                       #  规划过程参数
  obs_dis: 100

move_cmd:                               #  运动指令参数
